KsApi/fragments/ProjectFragment.graphql

```diff
@@ -6,6 +6,7 @@ fragment ProjectFragment on Project {
   category {
     ...CategoryFragment
   }
+  collaboratorPermissions
```

Comment:
This is the only property of our `MemberData` object that is currently supported by Graph. I will raise this to Paulo and request backend support.